C7/UIElements/Popups/PopupOverlay.cs

```diff
@@ -64,6 +88,18 @@ public void ShowPopup(Popup child, PopupCategory category)
 			log.Error("Invalid popup category");
 		}
 		AudioStreamWav wav = Util.LoadWAVFromDisk(Util.Civ3MediaPath(soundFile));
+
+		// 1. prevent mouse interaction with non-UI elements (ie. the map)
```

Comment:
Long term, need to learn how to use Godot UI properly. Couldn't find a more idiomatic solution on line for this particular problem though.

Short term, I plan to decouple UI from game logic as much as possible. This is kinda a prerequisite for decoupling game from server so that someday we could implement multiplayer.